.banzaicloud/charts/spotguide-zeppelin/charts/zeppelin-spark/README.md

```diff
@@ -0,0 +1,83 @@
+# Zeppelin-Spark Chart
+
+This is a composite chart deploying the following sub-charts:
+
+- [Zeppelin](https://github.com/banzaicloud/banzai-charts/tree/master/zeppelin)
+- [Spark](https://github.com/banzaicloud/banzai-charts/tree/master/spark) this is again a composite charts deploying the following sub-charts:
+  - [Spark History Server](https://github.com/banzaicloud/banzai-charts/tree/master/spark-hs) only if enabled (default *historyServer.enabled=true*)
+  - [Spark Resource Staging Server](https://github.com/banzaicloud/banzai-charts/tree/master/spark-rss) only if enabled (default *resourceStagingServer.enabled=false*)
+  - [Spark Shuffle Service](https://github.com/banzaicloud/banzai-charts/tree/master/spark-shuffle) only if enabled (default *sparkShuffle.enabled=false*)
+
+## Chart Details
+
+## Installing the Chart
+
+To install the chart:
+
+```
+$ helm install banzaicloud-stable/zeppelin
```

Comment:
Please specify how to add banzaicloud-stable repo to your helm

Reply:
will remove README files, as these are not standalone charts, they make sense only within the context of the spotguide.